### ISSUE TITLE
关于程序的一些个人问题

### ISSUE THREAD

**Issue by TreeOfEast** [JiaxiangBU/rumor_detection_2019_ncovEX#2]
作者君您好，在实现代码过程中，应该是由于官方数据变化的原因，训练集多加了几个label，所以想问一下，np.save( "../data/train_label.npy",train["label"].to_numpy())生成的文件中train_label指的是什么？现有的是
ncw_ label：取值为{0,1}，0表示需要进行真假判断，1表示不需要进行真假判断。
fake_ label：取值为{0,1},0表示非虚假新闻，1表示虚假新闻。
real_ label：取值为{0,1},0表示非真实新闻，1表示真实新闻。

**Comment by TreeOfEast** [comment]
如果我想利用模型进行虚假新闻检测，进行虚假新闻的筛选，应该修改哪些部分呢

**Comment by JiaxiangBU** [comment]
时间较远了，你可以关注下 gitbook，

```python
labels = np.array(df1[['ncw_label','fake_label','real_label']])
```

这里多分类处理。

然后我的 loss func

```
loss='categorical_crossentropy',
```

适配多任务的。

```
im_pred = keras.layers.Dense(3, activation="softmax", name="pred")(x)
```

计算图的 shape 是 3。

print 为

```
_________________________________________________________________ pred (Dense) (None, 3)
```

如果你是二分类，那么修改 3->2 即可，loss 你换成 binary 的交叉熵。

**Comment by JiaxiangBU** [comment]
时间较远了，你可以关注下 gitbook，

```python
labels = np.array(df1[['ncw_label','fake_label','real_label']])
```

这里多分类处理。

然后我的 loss func

```
loss='categorical_crossentropy',
```

适配多任务的。

```
im_pred = keras.layers.Dense(3, activation="softmax", name="pred")(x)
```

计算图的 shape 是 3。

print 为

```
_________________________________________________________________ pred (Dense) (None, 3)
```

如果你是二分类，那么修改 3->2 即可，loss 你换成 binary 的交叉熵。
